chapter_computational-performance/hardware_vn.md

```diff
@@ -139,8 +139,12 @@ CPUs have between 2 and 4 memory channels, i.e., they have between 40GB/s and 10
 Often there are two banks per channel. For instance AMD's Zen 3 Threadripper has 8 slots.
 -->
 
-*dịch đoạn phía trên*
-
+Bộ nhớ cơ bản nhất được sử dụng để lưu trữ dữ liệu khi cần truy cập một cách dễ dàng.
+Hiện tại bộ nhớ RAM CPU thường thuộc loại [DDR4](https://en.wikipedia.org/wiki/DDR4_SDRAM) mà mỗi mô-đun có băng thông 20-25GB/s.
+Mỗi mô-đun có đường truyền bus rộng 64 bit.
+Thông thường, các cặp mô-đun bộ nhớ cho phép sử dụng nhiều kênh.
+CPU có từ 2 đến 4 kênh bộ nhớ, nghĩa là, chúng có băng thông bộ nhớ tối đa từ 40GB/s đến 100GB/s.
+Thường có hai dải (_bank_) trên mỗi kênh. Ví dụ, Zen 3 Threadripper của AMD có 8 khe cắm.
```

Comment:
```suggestion
Thường có hai bank trên mỗi kênh. Ví dụ, Zen 3 Threadripper của AMD có 8 bank.
```
Theo https://www.amd.com/en/products/cpu/amd-ryzen-threadripper-3970x thì cái CPU trên có 4 mem channels => 8 banks nên `slots` là `banks` đúng ko nhỉ? ngoài ra em prefer ko dịch `bank`.

Reply:
Ông giáo @davidnvq có thảo luận gì thêm với điểm này không nhỉ?

Reply:
https://forums.tomshardware.com/threads/how-many-slots-in-a-bank.464356/
Mình cũng k chắc lắm, mà search qua thấy `bank` với `slot` khác nhau nhỉ. `bank` cover luôn cả (một vài ) slot

Reply:
Link @cuongvng  gửi chịu k thấy hai từ khóa kia :'(

Reply:
![image](https://user-images.githubusercontent.com/30251584/88286287-b9e72b00-cd1a-11ea-9bfb-d0e84b932ce9.png)

@davidnvq cuộn xuống specs thì thấy mem channels = 4, theo câu này thì mỗi channel có 2 bank. Nên em đoán câu sau `8 slots` là 8 banks.

Reply:
a thấy chắc k cần gộp bank với slot.

Reply:
ok để nguyên vậy.